### PR TITLE
GUACAMOLE-1289: Update Duo documentation for "Universal Prompt".

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: src/duo-auth.md [hunk 1]
```diff
@@ -110,57 +110,46 @@ The application-specific configuration information retrieved from Duo
...
+  the "API hostname" field. *This value is required.*
 
-duo-integration-key
+`duo-client-id`
 : The integration key provided for Guacamole by Duo. This value will
   have been generated by Duo when you added Guacamole as an "Web SDK"
   application, and can be found within the application details in the
-  "Integration key" field. *This value is required and must be EXACTLY
-  20 characters.*
+  "Client ID" field. *This value is required.*
```
Comment (necouchman) [inline src/duo-auth.md:125]: * Is this still an "integration key", or should this first line be changed to something like `The client identifier provided for Guacamole...`?
* While we're here modifying this block, perhaps change "an" to "a": `a "Web SDK"`

File: src/duo-auth.md [hunk 2]
```diff
@@ -110,57 +110,46 @@ The application-specific configuration information retrieved from Duo
...
 
-duo-secret-key
+  This value was formerly known as the "integration key" in older versions of
+  Duo's "Web SDK" and was configured with the `duo-integration-key` property
+  in older versions of Guacamole.
+
+`duo-client-secret`
 : The secret key provided for Guacamole by Duo. This value will have
```
Comment (necouchman) [inline src/duo-auth.md:132]: As with above - does it make sense to change this to `The client secret`?

**Review by necouchman — CHANGES_REQUESTED** [review]
Just a couple of potential changes...

File: src/duo-auth.md (hunk 1 above)
Comment (mike-jumper) [reply src/duo-auth.md:125]: Oops - will fix both of these.

**Comment by mike-jumper** [comment]
OK - I've looked through things and:

* Corrected all the wonky "an Web SDK" occurrences.
* Replaced old mentions of the integration and secret keys with their new names.

File: src/duo-auth.md [hunk 3]
```diff
@@ -110,57 +109,46 @@ The application-specific configuration information retrieved from Duo
...
 
-duo-secret-key
-: The secret key provided for Guacamole by Duo. This value will have
-  been generated by Duo when you added Guacamole as an "Web SDK"
+  This value was formerly known as the "integration key" in older versions of
+  Duo's "Web SDK" and was configured with the `duo-integration-key` property
+  in older versions of Guacamole.
+
+`duo-client-secret`
+: The shared secret provided for Guacamole by Duo. This value is will have
```
Comment (necouchman) [inline src/duo-auth.md:131]: Extra `is` in this line: `This value is will...`

File: src/duo-auth.md [hunk 4]
```diff
@@ -94,11 +94,10 @@ prompted by Duo for additional authentication:
 
 ![](images/duo-rename-guacamole.png)
 
-Once you've finished adding Guacamole as an "Web SDK" application, the
-configuration information required to configure Guacamole is listed
-within the application's "Details" section. You will need to copy the
-integration key, secret key, and API hostname - they will later be
-specified within `guacamole.properties`:
+Once you've finished adding Guacamole as a "Web SDK" application, the
+configuration information required to configure Guacamole is listed within the
```
Comment (necouchman) [inline src/duo-auth.md:98]: I wouldn't have worried about this if there weren't something else to change, but the `configuration information required to configure Guacamole` feels a bit redundant. Maybe just `the information required to configure Guacamole`?

**Review by necouchman — CHANGES_REQUESTED** [review]
A couple of more tweaks...

File: src/duo-auth.md (hunk 3 above)
Comment (mike-jumper) [reply src/duo-auth.md]: Fixed via rebase.

File: src/duo-auth.md (hunk 4 above)
Comment (mike-jumper) [reply src/duo-auth.md]: Fixed via rebase.

**Review by necouchman — APPROVED** [review]

**Pull request merged**